src/go/k8s/config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: vectorized/redpanda-operator
-  newName: vectorized/redpanda-operator
-  newTag: latest
+  newName: localhost/redpanda-operator
```

Comment:
won't this break ppl installing operator via kustomize? (see readme)

Reply:
This should not be needed as our kuttl tests is using different kustomization 
https://github.com/redpanda-data/redpanda/blob/e774bd4bbe51d6e485edf94c4f9a6e2438ad2701/src/go/k8s/kuttl-test.yaml#L16

https://github.com/redpanda-data/redpanda/blob/e774bd4bbe51d6e485edf94c4f9a6e2438ad2701/src/go/k8s/config/e2e-tests/kustomization.yaml#L7-L12

Reply:
Oh, I bet the Makefile doesn't use the test one. I'll take a look.

Reply:
reverted